firmware/port.c

```diff
@@ -140,7 +152,7 @@ void port_disable(PortData* p) {
 
 void port_send_status(PortData* p, u8 d) {
     if (p->reply_len >= BRIDGE_BUF_SIZE) {
-        bridge_disable_chan(p->chan);
+        port_error(p);
```

Comment:
So the reason you're reverting [these changes](https://github.com/tessel/t2-firmware/pull/103#discussion_r46512955) is to make the codebase more adaptable for another project? This will freeze up the coprocessor if the bridge state machine hits an undefined state, correct?